nncf/experimental/torch/fx/nncf_graph_builder.py

```diff
@@ -76,6 +113,11 @@ def create_nncf_graph(model: torch.fx.GraphModule) -> PTNNCFGraph:
         for source_node in model.graph.nodes:
             node_type, node_metatype = GraphConverter._get_node_type_and_metatype(source_node)
 
+            if node_metatype.get_subtypes():
```

Comment:
Looks like it should be part of _get_node_type_and_metatype

Reply:
And  it will be easier to add tests for this functions 😉

Reply:
And it will be easier to add tests for this functions 😉

Reply:
This will lead to regeneration of all torch fx dot files, you sure we really need this code style improvement for support of depthwise convolutions?

Reply:
I moved `get_subtype` to `_get_node_type_and_metatype`, hope it will do